#std_msgs/Header header
#uint32 height
#uint32 width
#sensor_msgs/PointField[] fields
#bool is_bigendian
#uint32 point_step
#uint32 row_step
#uint8[] data
#bool is_dense

sensor_msgs/PointCloud2 msg

#my-own add
uint32 kf_id
uint32 loop_id
